src/modules/claim/claim.dto.ts

```diff
@@ -124,10 +151,8 @@ export class NewClaimIssueDTO extends ClaimIssueDTO {
   claimTypeVersion: string;
 
   @IsEnum(RegistrationTypes, { each: true })
-  // Optional so as to not break existing clients. Can be made mandatory in future.
-  @IsOptional()
   @ApiProperty()
-  registrationTypes: [RegistrationTypes];
+  registrationTypes: [RegistrationTypes, RegistrationTypes?];
```

Comment:
Isn't `RegistrationTypes?` superset of `RegistrationTypes`. I.e. it is `RegistrationTypes` + `undefined\null`?
```suggestion
  registrationTypes: [RegistrationTypes?];
```

Reply:
It;s ideally present how the object should look like, at lest one element is required but array can contain max two elements. Previously we can have a object with a five registration types, which is not allowed, currently TS will catch this situation.